test/e2e/test/logstash/checks.go

```diff
@@ -65,29 +85,63 @@ func CheckStatus(b Builder, k *test.K8sClient) test.Step {
 }
 
 func (b Builder) CheckStackTestSteps(k *test.K8sClient) test.StepList {
-	println(test.Ctx().TestTimeout)
 	return test.StepList{
-		{
-			Name: "Logstash should respond to requests",
-			Test: test.Eventually(func() error {
-				client, err := NewLogstashClient(b.Logstash, k)
-				if err != nil {
-					return err
-				}
-				bytes, err := DoRequest(client, b.Logstash, "GET", "/")
-				if err != nil {
-					return err
-				}
-				var status logstashStatus
-				if err := json.Unmarshal(bytes, &status); err != nil {
-					return err
-				}
+		b.CheckMetricsRequest(k,
+			Request{
+				Name: "metrics",
+				Path: "/",
+			},
+			Want{
+				Status: "green",
+			}),
+		b.CheckMetricsRequest(k,
+			Request{
+				Name: "default pipeline",
+				Path: "/_node/pipelines/demo",
```

Comment:
```suggestion
				Path: "/_node/pipelines/main",
```

This needs to be `main` to match the value in the samples file